platformio_esp82xx_envs.ini

```diff
@@ -44,7 +44,7 @@ platform_packages         = ${regular_platform.platform_packages}
 build_flags               = ${regular_platform.build_flags} 
                             ${esp8266_4M1M.build_flags} 
                             -DPLUGIN_BUILD_CUSTOM
-lib_ignore                = LittleFS, ESP32_ping, ESP32WebServer, ESP32HTTPUpdateServer, Adafruit GFX Library, LOLIN_EPD, Adafruit ILI9341, Adafruit BusIO, Adafruit NeoPixel, Adafruit Motor Shield V2 Library, RN2xx3 Arduino Library
+lib_ignore                = LittleFS, ESP32_ping, ESP32WebServer, ESP32HTTPUpdateServer, Adafruit GFX Library, LOLIN_EPD, Adafruit ILI9341, Adafruit BusIO, Adafruit NeoPixel, Adafruit Motor Shield V2 Library, RN2xx3 Arduino Library, HeatpumpIR
```

Comment:
This change makes it impossible to build the HeatpumpIR in the `env:custom_IR_ESP8266_4M1M` env.

Reply:
I know, but for this PR the library HeatpumpIR ist not necessary.
I found no other way to avoid the linking error (*.elf -> file not found)

Reply:
Well if it is merged, it is no longer just about this PR.
I will have a look to see what I can do here.